Promact.Oauth.Server/src/Promact.Oauth.Server/Services/MessageServices.cs

```diff
@@ -46,18 +45,17 @@ public Task SendEmailAsync(string email, string subject, string message)
             {
                 using (var smtp = new SmtpClient())
                 {
-                    _logger.LogInformation("Start Email Sending");
+                  
                     smtp.ConnectAsync(_appSettings.Value.Host, _appSettings.Value.Port, MailKit.Security.SecureSocketOptions.None).Wait();
                     smtp.AuthenticateAsync(credentials: new NetworkCredential(_appSettings.Value.UserName, _appSettings.Value.Password)).Wait();
-                    smtp.SendAsync(msg, CancellationToken.None).Wait();
-                    smtp.DisconnectAsync(true, CancellationToken.None).Wait();
+                    smtp.SendAsync(msg).Wait();
```

Comment:
Now change removed Async method , used the synchronous method
 smtp.Connect(_appSettings.Value.Host,_appSettings.Value.Port,MailKit.Security.SecureSocketOptions.None);
 smtp.Authenticate(credentials: new NetworkCredential(_appSettings.Value.UserName, _appSettings.Value.Password));
 smtp.Send(msg, CancellationToken.None);
 smtp.Disconnect(true, CancellationToken.None);